src/state/profile/helpers.ts

```diff
@@ -111,7 +111,8 @@ export const getProfile = async (address: string): Promise<GetProfileResponse> =
     } as Profile
 
     return { hasRegistered, profile }
-  } catch (error) {
+  } catch (e) {
```

Comment:
Better error handling - part of my debugging why the /collectibles page was crashing.